.github/workflows/build.yml

```diff
@@ -2,8 +2,6 @@ name: Release build
 
 on:
   push:
-    branches:
-      - "release/**"
```

Comment:
instead I would add `pull_request:` (with no value -- which is the same as `pull_request: null`) otherwise it won't run for external PRs